.clang-format

```diff
@@ -15,3 +16,28 @@ IncludeBlocks: Regroup
 IndentCaseLabels: true
 IndentPPDirectives: AfterHash
 NamespaceIndentation: All
+
+---
+Language: ObjC
```

Comment:
for some reason visual studio thinks that c++/cli is objC. this makes the formatting consistent.